src/main/java/dev/j3fftw/litexpansion/weapons/NanoBlade.java

```diff
@@ -33,12 +33,12 @@ public ItemUseHandler getItemHandler() {
         return event -> {
             ItemMeta nanoBladeMeta = event.getItem().getItemMeta();
             final Enchantment enchantment = Enchantment.getByKey(Constants.NANO_BLADE_ACTIVE_ENCHANT);
+            final boolean removedLevel = nanoBladeMeta.removeEnchant(enchantment);
```

Comment:
I probably wouldn't bother making a variable, just stick the removeEnchant in the if